www/js/app.js

```diff
@@ -92,9 +91,22 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
     });
     $('#prefix').on('keyup', function(e) {
         if (selectedArchive !== null && selectedArchive.isReady()) {
+            if (e.which === 27) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which seems to be deprecated.
Could you try to use https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key instead? (Check the example at the end of this web page, because IE uses specific values for this property, that need to be handled too)

Reply:
Ah, OK, that's good to know. I'll try the new method, and will check the Edge / IE workarounds.

Reply:
Done. Let's see if that makes a difference with the tests. I tested manually on Edge 44, Edge 76 (it's incremented by one recently), Firefox ESR and IE11.